.travis.yml

```diff
@@ -6,8 +6,8 @@ android:
   components:
     - tools
     - platform-tools
-    - build-tools-26.0.2
-    - android-26
+    - build-tools-27.0.3
```

Comment:
```suggestion
    - build-tools-28.0.3
```

Reply:
Yeah, I just realized that as I submitted the 4th commit lol